docs/Support/glossary.md

```diff
@@ -14,10 +18,28 @@ In cryptography, a digital signature proves that a user controls the private key
 In working with the Lit network, someone will sign with their wallet when they want to mint a PKP. That is a transaction signature — where someone is sending a transaction (minting). This signature is then used as the method of authentication over the PKP and associated Lit Actions.
 ### **Distributed Key Generation**
 A cryptographic process in which key generation requires participation from above a certain threshold of nodes, where each node only holds a ******share****** of the complete keypair.
+### **[Elliptic Curve Digital Signature Algorithm (ECDSA)](https://blog.cloudflare.com/ecdsa-the-digital-signature-algorithm-of-a-better-internet/)**
+A cryptographic algorithm used by Ethereum to ensure that funds can only be spent by their owners. It's the preferred method for creating public and private keys, relevant for account [address](https://ethereum.org/en/glossary/#address) generation and [transaction](https://ethereum.org/en/glossary/#transaction) verification. Can be used for signing messages, but not for encryption purposes.
```

Comment:
I like the definition of ECDSA here: 

"The [ECDSA](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm) (Elliptic Curve Digital Signature Algorithm) is a cryptographically secure digital signature scheme, based on the elliptic-curve cryptography" - https://cryptobook.nakov.com/digital-signatures/ecdsa-sign-verify-messages

Additional explanation: "ECDSA is mainly used for digital signatures. A digital signature is an authentication method used where a public key pair and a digital certificate are used as a signature to verify the identity of a recipient or sender of information."